lib/backend.js

```diff
@@ -26,10 +26,17 @@ async function basicNPCOrders (config) {
           roomName: terminal.room
         }
       }
-      buy.remainingAmount = market.buyAmount
-      buy.totalAmount = market.buyAmount
-      buy.amount = market.buyAmount
-      buy.price = market.buyPrices[mineral]
+      if(market.priceMode === 'fixed') {
+        buy.remainingAmount = market.buyAmount
+        buy.totalAmount = market.buyAmount
+        buy.amount = market.buyAmount
+        buy.price = market.buyPrices[mineral]
+      } else {
```

Comment:
My only thoughts are make the `else` an `else if (market.priceMode === 'random')`, this leaves it open to other modes in the future without breaking people's configs.

Reply:
I can see that! I agree.